GeneralsMD/Code/GameEngine/Source/Common/RTS/Player.cpp

```diff
@@ -462,6 +462,7 @@ void Player::init(const PlayerTemplate* pt)
 			{
 				m_money.deposit( TheGlobalData->m_defaultStartingCash.countMoney(), FALSE );
 			}
+			m_money.setStartingCash(m_money.countMoney());
```

Comment:
This needs another look. First depositing money and then setting starting cash is not good.

It should be one call.

Better design `setStartingCash` in a way that prior calls to deposit and similar are not necessary.